src/CreeDictionary/API/search/runner.py

```diff
@@ -24,6 +25,10 @@ def search(
         query=query, include_auto_definitions=include_auto_definitions
     )
 
+    if search_run.query.eip:
+        eip_search = EipSearch(search_run)
+        eip_search.analyze_query()
```

Comment:
Am I understanding this correctly?

Phase 1 parses the query, **mutating it** so that the the query string contains "uninflected" forms... [continued in a later comment]